### ISSUE TITLE
[BUG] failure on CENT8/RHEL8

### ISSUE THREAD

**Issue by noelmcloughlin** [saltstack-formulas/docker-formula#243]
### Pillar / config used

---

### Describe the bug

The following state failed (salt 3000).
```
          ID: docker-package
    Function: pkg.installed
        Name: docker-ce
      Result: False
     Comment: Error occurred installing package(s). Additional info follows:

              errors:
                  - Running scope as unit: run-r232395d2686a4b8f8d0907ff0be509f7.scope
                    CentOS Centos linux 8 (core) Docker Package Rep 477  B/s | 576  B     00:01
                    Failed to download metadata for repo 'docker-ce-stable'
                    Error: Failed to download metadata for repo 'docker-ce-stable'
```

**Comment by JustinZ** [comment]
It seems it doesn't support docker ce ?

**Comment by noelmcloughlin** [comment]
Probably. CentOS is disabled in Travis CI/CD. I got #255  working for CentOS7 but I'm not sure about CentOS8.

**Comment by noelmcloughlin** [comment]
Fixed in merged #256
